nion/swift/resources/menu_config.json

```diff
@@ -41,12 +41,23 @@
                 "action_id": "file.import_data"
             },
             {
-                "type": "item",
-                "action_id": "file.export"
-            },
-            {
-                "type": "item",
-                "action_id": "file.export_svg"
+                "type": "sub_menu",
+                "menu_id": "file_export",
+                "title": "Export...",
```

Comment:
Sub-menus should not have "..." on them.  But I don't think these should go into a sub-menu, at least for now.

As part of this change, though, we could update the context menu to supply "single export" and "export svg". That's done here: https://github.com/nion-software/nionswift/blob/6f71ebc70e61ea209ba17f7a7e34939cd8f0aeed/nion/swift/DocumentController.py#L2641

Reply:
Hi Chris, Can I ask why we wouldn't want to group these in a sub menu?

Reply:
I'm not 100% opposed to it, but the File menu is relatively short, so adding an item isn't a big change. The downside of the sub menu is it is (a) less visible to users exploring the menus (b) one extra click/mouse-motion to select.

I actually think it might be nice to have them all visible with a separator between the 'Import' items and the 'Export' items. This would be the addition of two lines in the menu.

If your group has a strong opinion, it's fine. But if not, let's go with no sub-menu with the option of putting them in a sub-menu in another PR.